src/registry/templates/registry.json

```diff
@@ -0,0 +1,627 @@
+{
```

Comment:
I can't really follow the structure of the schema here.  Sometimes a service is the top level key, sometimes a language.  What's the structural intent here?

I would think a flatter list of `template name => info` would be easier, which can then have tags for what services/languages it includes.  That's then simple enough to filter on if we want just the templates that include mariadb or just the templates that use node, for example.

Reply:
Also... should this file live rather in template-builder rather than docs?  It seems like we could auto-generate it off of template-builder itself just by parsing the YAML files.  (Famous last words, I know...)

Reply:
The use of "templates" here is a namespace conflict with the actual templating system; they are not necessarily connected. I believe everything is relocated under the `images/` path now, which seems better to me.

These should not live in template-builder, which is _our implementation_ of the catalog and templating system. This registry is intended to be a system-wide registry of all available apps and services (this is why some of the languages are keys in the file, those are just types of application containers) and is not formally connected to template-builder or the templating process.

Reply:
Ah, I see, the registry.json in `templates/` is different and serves as an index of which templates consume which service types.

Reply:
Yes, this will be removed from this PR for now.